### PR TITLE
Pyramid and javascript/css bundling

### PR DESCRIPTION
**Pull request by ergo**
Explain how to create frontend build pipeline.

### CODE REVIEW

File: docs/static_assets/files.rst [hunk 1]
```diff
@@ -189,3 +189,161 @@ request object as a ``cgi.FieldStorage`` object accessible through the
...
+
+Now you can configure/run your frontend pipeline with webpack/gulp/rollup or other solution.
+
+If you run the command:
+
+::
+
+    yourapp_build_statics development.ini
+
+it will start the build process and its result will be a fresh `static` directory in same
```
Comment (stevepiercy) [inline docs/static_assets/files.rst]: Capitalize "It"

Use double backticks:

```
``static``
```

**Comment by stevepiercy** [comment]
@ergo In addition to this, could you generate an example project and repository using https://github.com/Pylons/pyramid-cookiecutter-starter and your setup?

**Comment by ergo** [comment]
@stevepiercy I've made all the changes you requested.

**Comment by ergo** [comment]
@stevepiercy Here is an example project https://github.com/ergo/pyramid-bundling-example - you will need `nodejs` 8.x and `yarn` to run the build process for front assets.

**Comment by stevepiercy** [comment]
I'll push some changes to the demo repo https://github.com/Pylons/pyramid-bundling-example

**Comment by stevepiercy** [comment]
Thanks, @ergo!

**Pull request merged**